### DISCUSSION TITLE
[FR] Make it possible to copy/paste CKEditor content containing nested entries, across different fields/editor instances

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/ckeditor#237] [Ideas] [answered]
It'd be great if it was possible to copy/paste nested entries between different CKEditor fields/editor instances.

Currently, if authors attempt to copy the contents of a CKEditor field into a different CKEditor field, _if the field contains nested entries_ the paste operation will fail with an error message "Entries cannot be copied between CKEditor fields.". 

I realise this is working as intended (as per #192), but from authors POV this behavior is difficult to understand – i.e., why can they sometimes copy/paste rich text between fields, and sometimes not?

The current behavior is also frustrating in that it severely limits portability of rich text content, whenever nested entries are involved. Authors generally think of the entire CKEditor field's contents as a single blob of text, and generally expect to be able to copy and paste that text across different fields, entries or even sites, regardless of what the text contains or not. I don't think that's an unreasonable expectation 🙂

IMO, the most intuitive behavior when pasting content containing nested entries into a CKEditor field, would be that Craft simply duplicated the nested entries to the target entry/field/site.

**Comment by brandonkelly** [comment, marked as answer]
We made this possible in CKEditor 4.1.0 🎉  (#244)
